header/header_modifier.go

```diff
@@ -27,31 +27,52 @@ func init() {
 	parse.Register("header.Modifier", modifierFromJSON)
 }
 
-type modifier struct {
+type HeaderSetBehavior int
+
+const (
+	ReplaceValues HeaderSetBehavior = iota
+	AppendValues
+)
+
+type Modifier struct {
```

Comment:
Add comment for exported type
https://github.com/golang/go/wiki/CodeReviewComments#doc-comments